src/main.rs

```diff
@@ -292,9 +272,11 @@ fn transform(args: Transform) -> Result<()> {
     let data = fs::read(&args.input)?;
     let mut ir: IR = serde_yaml::from_slice(&data)?;
     let config = load_config(&args.transform)?;
-    for t in &config.transforms {
-        info!("running: {:?}", t);
-        t.run(&mut ir)?;
+    if let Some(transforms) = &config.transforms {
```

Comment:
the `chiptool transform` command should also expand the includes, same as `chiptool generate`.

Reply:
Fix this.